src/index.d.ts

```diff
@@ -1,3 +1,5 @@
+import Peer from 'simple-peer-light'
```

Comment:
I just noticed that this is causing `import` issues with TypeScript. I'll either need to use `simple-peer` types or change line 55 to use `any` instead of `Peer`.

Please hold off on merging this until I've sorted this out. I will follow up once I've got a solution!